core/client/src/blockchain.rs

```diff
@@ -258,3 +257,18 @@ pub fn tree_route<Block: BlockT, Backend: HeaderBackend<Block>>(
 		pivot,
 	})
 }
+
+/// A list of all well known keys in the blockchain cache.
+pub mod well_known_cache_keys {
```

Comment:
I think the `client` is the wrong place. We should not try to put more and more stuff into client. I think `primitives` is the correct place for these constants + types.

Reply:
Could do, but the cache itself is declared in client. And not in primitives. So I thought it is better to keep them together. What do you think?

Reply:
Hmm yeah, that is maybe something that can be done when the cache is extracted from the client crate.